Makefile

```diff
@@ -45,10 +45,8 @@ docs-url-alive-check:
 
 .PHONY: \
 	all \
-	binary-tarball \
+	kata-tarball \
+	install-tarball \
 	default \
-	install-binary-tarball \
```

Comment:
@amshinde the commit message is a little unclear, were these targets supposed to be added to the tools/packaging/kata-deploy/local-build Makefile?

Reply:
I think it should matter either way since this Makefile has an include for the other one. I added in the top level Makefile for them to be easily discoverable from end user point of view.